### DISCUSSION TITLE
How to know which job was connected to which runner?

### DISCUSSION THREAD

**Discussion by dongho-jung** [actions/actions-runner-controller#1987] [General]
I'm monitoring each runners' resource metric.

However for now, If I see a weird metric of runner and want to find what was that, then I have to crawl all over repos and actions in my org.

Maybe there was already neat way for this though, I couldn't find yet.

Or I suggest that can we have some additional information about which repo and job was connected to this runner in [startup.sh](https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/startup.sh)

**Comment by dongho-jung** [comment]
Sorry it was my fault. I overlooked them. Indeed it already did its job nicely 

<img width="827" alt="image" src="https://user-images.githubusercontent.com/19762154/200218962-6e01ab84-c6fe-48e9-b2d0-c27a1fd01950.png">

but If you have a lot of jobs that have the same name of jobs in other repos, you should rename your jobs to identify which job is which I think

**Reply by toast-gear** [reply]
acitons/runner should be enhanced to include the repository and or workflow name as part of its output. Perhaps it's worth raising an issue in actions/runner?

**Reply by dongho-jung** [reply]
I think it's worth raising an issue. If I can contribute, I will. It would be helpful when you track down which job consumes a lot of cluster resource or other scenarios.
